src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+}
+
+form {
+  margin-top: 20px;
+}
+
+.search {
+  font-family: inherit;
+  font-weight: bold;
+  color: #3d4e61;
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  background-size: 19px 19px;
+  background-position: 2% 50%;
+  padding-left: 62px;
+  outline: none;
+}
+
+.search--small {
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  background-size: 11px 11px;
+  background-position: 1% 50%;
+  padding-left: 33px;
+}
+
+.search::placeholder {
+  font-family: inherit;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search--small::placeholder {
+  font-style: normal;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
Remove redundant styles. If we already have the main styles for the class before, for a certain state of the class, it is not necessary to write the same styles, only different values if we want to change or add something new. Fix it with other pseudo-classes too and remove redundant code
```suggestion
```